### ISSUE TITLE
Failed to activate the appcelerator-titanium package

### ISSUE THREAD

**Issue by prashantsaini1** [appcelerator/atom-appcelerator-titanium#185]
### Description
I just installed the package afresh and restarted the Atom but the package did not reflect for Titanium project. Then I disabled the package and enabled it again where it shows this filed bug.

### Steps to reproduce
![Screenshot 2019-10-09 at 10 16 16 PM](https://user-images.githubusercontent.com/20779626/66502661-5bcabd80-eae3-11e9-84f4-d433b5637f27.png)
1. Open any Titanium app project in Atom.
2. Install the appcelerator-titanium package.
3. Restart Atom
4. Nothing is shown even after a fresh install or restart.
5. Disable the package and enable it again shows this filed bug.

### Expected Behavior
The package should be started normally.

### Environment
| Software | Version |
| ---------------- | ---------- |
| Titanium SDK | 7.5.2.GA or just any version|
| CLI | 7.1.1|
| Atom | Latest - 1.40.1|
| Titanium Atom Package | 1.9.1|

### Logs (Seen after step-5)
```
TypeError: Arguments to CompositeDisposable.add must have a .dispose() method
    at assertDisposable (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1212754)
    at CompositeDisposable.add (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1213722)
    at Object.consumeProvider (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:162477)
    at Object.consumeProvider_4 (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:162285)
    at Provider.provide (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:1126065)
    at ServiceHub.provide (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3290410)
    at Package.activateServices (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3298696)
    at Package.activateNow (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3295672)
    at measure (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3294996)
    at Package.measure (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3292606)
    at activationPromise.activationPromise.Promise (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3294856)
    at new Promise (<anonymous>)
    at Package.activate (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3294799)
    at PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:382285)
    at o.forEach.e (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:377451)
    at Array.forEach (<anonymous>)
    at disabledPackagesSubscription.disabledPackagesSubscription.config.onDidChange (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:377435)
    at emitter.on (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:334486)
    at Function.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1209208)
    at Emitter.emit (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:1210649)
    at Config.emitChangeEvent (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:336974)
    at Config.setRawValue (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:334214)
    at Config.set (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:330008)
    at Config.removeAtKeyPath (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:332031)
    at Package.enable (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3292391)
    at PackageManager.enablePackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:375149)
    at HTMLButtonElement.enablementButtonClickHandler (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:14:2602116)
```

**Comment by ewanharris** [comment]
@prashantsaini1, I just want to confirm is this error occurring for you constantly and stopping the package from loading, or if you enable the package and then reload the Atom window does it still work?

I can reproduce this and I think it's down to us potentially not clearing up a autocomplete provider or similar, but if I enable the package and then reload Atom then I can use the package as normal

**Comment by prashantsaini1** [comment]
The behaviour is that if I launch Atom afresh, the package does not recognise the project and I am unable to see the package related bar in Atom. This force me to disable and enable the package and that's there it throws the error.

**Comment by ewanharris** [comment]
@prashantsaini1, I'm not sure that there's anything we can do here, I've debugged the error and it looks to be down to some internal mishandling in Atom. I get the same behaviour if I disable and re-enable any of the builtin autocomplete packages. There's a bug report for the autocomplete-plus package over at https://github.com/atom/autocomplete-plus/issues/851.

When I reproduce this error, if I set the package back to enabled and then fully quit Atom and reopen a Titanium project I'm able to use the package just fine. I see the commands in the command palette and the toolbar loads (it's possible you might need to toggle it back using `Appc: Toggle Toolbar`).
